buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1389,5 +1402,39 @@ artifactId=${project.name}
         args argsNeeded
       }
     }
+
+
+    /** ***********************************************************************************************/
+
+    // Method to create the PortableValidatesRunnerTask.
+    project.ext.createPortableValidatesRunnerTask = {
+      def config = it ? it as PortableValidatesRunnerConfig : new PortableValidatesRunnerConfig()
+      def name = config.name ? config.name : "validatesPortableRunner"
+      def testCategories = {
+        includeCategories 'org.apache.beam.sdk.testing.ValidatesRunner'
```

Comment:
Presumably these categories will be runner-specific (with the exception of ValidatesRunner itself). Can you add the excluded categories as a configuration parameter that is specified when calling `applyJavaNature`?

Reply:
Sorry, make that `createPortableValidatesRunnerTask` rather than `applyJavaNature`.

Reply:
Runner can provide test categories in config.testCategories so they can control the appropriate test category.
Should I add only the exclude categories in the parameters?